mycroft/audio/stt.py

```diff
@@ -0,0 +1,56 @@
+from mycroft.api import STTApi, HTTPError
```

Comment:
Why is this moved from speech module?

Reply:
the aim is to deprecate the whole "client" folder here

the decision to move it into audio came from a discussion with @forslund some time ago in mattermost, i believe he intended to do something similar

@forslund pinging you because your feedback is appreciated about this change